auditlog_tests/tests.py

```diff
@@ -600,6 +601,18 @@ class DateTimeFieldModelTest(TestCase):
     utc_plus_one = timezone.get_fixed_timezone(datetime.timedelta(hours=1))
     now = timezone.now()
 
+    def setUp(self):
+        super().setUp()
+        self._context = warnings.catch_warnings()
+        self._context.__enter__()
+        warnings.filterwarnings(
+            "ignore", message=".*naive datetime", category=RuntimeWarning
+        )
+
+    def tearDown(self):
+        self._context.__exit__()
+        super().tearDown()
+
```

Comment:
is it possible to just ignore the warning for that specific test that created the warning?

Reply:
This warning is caused by multiple tests in this test class. I decided to effectively wrap every method in this class in `with catch_warnings`.